platform/ios/PolyPodApp/PodApi/Triplestore/Triplestore.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import MessagePack
+import PolyPodCoreSwift
+
+protocol TriplestoreProtocol: AnyObject {
```

Comment:
This could be called Tripplestore, and instead of having a new `Triplestore` class, consider just extending Core:

```swift
extension Core: TripleStore {
  func query(
         query: String,
         completionHandler: @escaping (MessagePackValue, Error?) -> Void
     ) {...}
     
     func update(
         query: String,
         completionHandler: @escaping (Error?) -> Void
     ){...}
}
```